modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -49,20 +49,20 @@ Certificate files are typically stored as Base64 files, with extensions such as
 ** `openssl x509 -in cert.pem -out cert.cer`
 ====
 
-. Create a new ConfigMap with the required TLS certificates:
+. Create a new `ConfigMap` object with the required TLS certificates:
 +
 [subs="+attributes,+quotes"]
 ----
 $ {orch-cli} create configmap custom-certs --from-file=__<bundle_file_path>__ -n={prod-namespace}
 ----
 +
-To apply more than one bundle, add another `-from-file=_<bundle_file_path>_`. Alternatively, create another ConfigMap.
+To apply more than one bundle, add another `-from-file=_<bundle_file_path>_`. Alternatively, create another `ConfigMap` object.
 
-. Label created ConfigMaps with the `app.kubernetes.io/part-of=che.eclipse.org` and `app.kubernetes.io/component=ca-bundle` labels:
+. Label the created `ConfigMap` objects with the `app.kubernetes.io/part-of=che.eclipse.org` and `app.kubernetes.io/component=ca-bundle` labels:
 +
 [subs="+attributes,+quotes"]
```

Comment:
```suggestion
[source,terminal,subs="+attributes,+quotes"]
```

Reply:
To sum up Robert's feedback, we don't use `source,terminal` because of these reasons:
1. The tooltip `Terminal` is confusing to readers because `Terminal` is not a language.
2. Regarding the convenience of the copy-paste tooltip button: We write `$` at the beginning of commands, so `$` also gets copy-pasted, requiring a manual edit of the copy-pasted text, as opposed to simply selecting it with a mouse and pasting it. This can mess things up when users paste it onto a command line, which is where indeed they should be pasting it.
3. Regarding the convenience of the copy-paste tooltip button: the copy-paste tooltip button is only a feature in the Che docs, and it is absent from the Dev Spaces docs.